docs/tutorials/fundamentals/viewer.md

```diff
@@ -314,21 +340,24 @@ viewer.add_image(blobs, name='blobs', opacity=0.5, colormap='red')
 nbscreenshot(viewer, alt_text="A 2d view of the moon on top of which is overlaid a 3d volume containing blobs through which you can navigate using the dimension slider.")
 ```
 
-In order to get or to update the current position of the slider, use 
+In order to get or to update the current position of the slider, use:
 
-```viewer.dims.current_step```
-
-that is in integer steps. 
-
-`viewer.dims.point` contains the position in world coordinates (ie including scale and translate transformations)
+```python
+# to get the current position
+viewer.dims.current_step
+# to change the current position
+viewer.dims.current_step = 3
+```
 
+`viewer.dims.point` contains the position in world coordinates (i.e., including
+scale and translate transformations).
 
 ### Viewer buttons
 
-Underneath the layers list there is a row of buttons that includes the `Console`
-button that will show or hide our console that allows you to interact with a
-python kernel. Inside the console you can access the viewer using the `viewer`
-argument.
+Underneath the layers list there is a row of buttons. The first is the `Console`
+button that will show or hide our console, allowing you to interact with a
+python kernel. Inside the console you can access the
+{class}`Viewer<napari.Viewer>` instance using the `viewer`.
 
 When the console button is clicked, the console will appear at the bottom of the
```

Comment:
Should we add a note that the console is disabled when the viewer is opened from ipython/jupyter in favor of the user continuing the use of those interactive consoles?
See user confusion for example here https://github.com/napari/napari/issues/5211

Reply:
Good idea, I didn't know this!